app/models/bourbon.rb

```diff
@@ -0,0 +1,10 @@
+class Bourbon < ActiveRecord::Base
+  belongs_to :user
+
+  validates :name, presence: true
+  validates :name, uniqueness: true
+  validates :proof, presence: true
+  validates :proof, numericality: { :greater_than => 0,
+    :less_than_or_equal_to => 200 }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.